### PR TITLE
IGAPP-879: Remove logging statements

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Comment by klinzo** [comment]
Testing:
* sending to sentry: (`yarn build`), running the app (`cd dist/integreat` and `python3 -m http.server`) and throwing an test error somewhere in the used files/routes. Then you can verify the error was actually sent to sentry
* logging for developer friendly builds: `yarn start` and look at console

File: web/src/utils/sentry.ts [hunk 1]
```diff
@@ -1,52 +1,84 @@
...
+import { FetchError, NotFoundError } from 'api-client'
+
 import buildConfig from '../constants/buildConfig'
 
-const loadSentry = async (): Promise<typeof Sentry> =>
-  import(
-    /* webpackChunkName: "sentry" */
-    '@sentry/react'
-  )
+const sentryEnabled = (): boolean => buildConfig().featureFlags.sentry
```
Comment (SteffiStuffel) [inline web/src/utils/sentry.ts:8]: Why not just `const sentryEndable = buildConfig().featureFlags.sentry`?

File: web/tools/webpack.config.ts [hunk 2]
```diff
@@ -87,7 +87,7 @@ const createConfig = (
   // eslint-disable-next-line no-console
   console.log('Used config: ', buildConfigName)
   // eslint-disable-next-line no-console
-  console.log('Configured as running in dev server: ', !devServer)
+  console.log('Configured as running in dev server: ', devServer)
```
Comment (SteffiStuffel) [inline web/tools/webpack.config.ts:90]: Why this change?

**Review by SteffiStuffel — APPROVED** [review]
Could not make it to send a error to sentry.

I added `reportError(new Error('Sentry test of IGAPP-879'))` in the code
did `yarn build` + `cd dist/integreat` + `python3 -m http.server`

File: web/tools/webpack.config.ts (hunk 2 above)
Comment (klinzo) [reply web/tools/webpack.config.ts:90]: Because it was wrong before, it said `Configured as running in dev server: true` for production builds.

File: web/src/utils/sentry.ts (hunk 1 above)
Comment (klinzo) [reply web/src/utils/sentry.ts:8]: Was that way before, not sure why it was done that way. However, it makes mocking the build config way easier in tests since you can change values in between test. If this is not a function, it keeps the first value (at least that happened to me on native lately).

File: web/src/routes/PoisPage.tsx [hunk 3]
```diff
@@ -49,8 +49,6 @@ const PoisPage = ({ cityCode, languageCode, cityModel, pathname, languages }: Ci
   const formatter = useContext(DateFormatterContext)
   const { viewportSmall } = useWindowDimensions()
   const navigate = useNavigate()
-  // eslint-disable-next-line no-console
-  console.log('To use geolocation in a development build you have to start the dev server with\n "yarn start --https"')
```
Comment (f1sh1918) [inline web/src/routes/PoisPage.tsx:53]: why did you rm this hint?

**Review by f1sh1918 — APPROVED** [review]

File: web/src/routes/PoisPage.tsx (hunk 3 above)
Comment (klinzo) [reply web/src/routes/PoisPage.tsx:53]: Will readd it if the build is developer friendly
